shared/src/main/scala/uk/gov/homeoffice/drt/arrivals/Arrival.scala

```diff
@@ -122,24 +120,31 @@ case class Arrival(Operator: Option[Operator],
 
   val bestPcpPaxEstimate: TotalPaxSource = {
     val matchedTotalPax = TotalPax match {
-      case totalPax if totalPax.exists(tp => tp.feedSource == ScenarioSimulationSource && tp.pax > 0) =>
-        totalPax.find(tp => tp.feedSource == ScenarioSimulationSource)
-      case totalPax if totalPax.exists(tp => tp.feedSource == LiveFeedSource && tp.pax > 0) =>
-        totalPax.find(tp => tp.feedSource == LiveFeedSource && tp.pax > 0)
-      case totalPax if totalPax.exists(tp => tp.feedSource == ApiFeedSource && tp.splitSource.contains(ApiSplitsWithHistoricalEGateAndFTPercentages) && tp.pax > 0) =>
-        totalPax.find(tp => tp.feedSource == ApiFeedSource && tp.splitSource.contains(ApiSplitsWithHistoricalEGateAndFTPercentages))
-      case totalPax if totalPax.exists(tp => tp.feedSource == ForecastFeedSource && tp.pax > 0) =>
-        totalPax.find(tp => tp.feedSource == ForecastFeedSource && tp.pax > 0)
-      case totalPax if totalPax.exists(tp => tp.feedSource == ApiFeedSource && tp.splitSource.contains(Historical) && tp.pax > 0) =>
-        totalPax.find(tp => tp.feedSource == ApiFeedSource && tp.splitSource.contains(Historical) && tp.pax > 0)
-      case totalPax if totalPax.exists(tp => tp.feedSource == ApiFeedSource && tp.pax > 0) =>
-        totalPax.find(tp => tp.feedSource == ApiFeedSource && tp.pax > 0)
-      case totalPax if totalPax.exists(tp => tp.feedSource == AclFeedSource && tp.pax > 0) =>
-        totalPax.find(tp => tp.feedSource == AclFeedSource)
+      case totalPax if totalPax.exists(tp => tp.feedSource == ScenarioSimulationSource && tp.pax.isDefined) =>
+        excludeTransferPax(totalPax.find(tp => tp.feedSource == ScenarioSimulationSource))
+      case totalPax if totalPax.exists(tp => tp.feedSource == LiveFeedSource && tp.pax.isDefined) =>
+        excludeTransferPax(totalPax.find(tp => tp.feedSource == LiveFeedSource && tp.pax.isDefined))
+      case totalPax if totalPax.exists(tp => tp.feedSource == ApiFeedSource && tp.pax.isDefined) =>
+        totalPax.find(tp => tp.feedSource == ApiFeedSource && tp.pax.isDefined)
+      case totalPax if totalPax.exists(tp => tp.feedSource == ForecastFeedSource && tp.pax.isDefined) =>
+        excludeTransferPax(totalPax.find(tp => tp.feedSource == ForecastFeedSource && tp.pax.isDefined))
+      case totalPax if totalPax.exists(tp => tp.feedSource == HistoricApiFeedSource && tp.pax.isDefined) =>
+        totalPax.find(tp => tp.feedSource == HistoricApiFeedSource && tp.pax.isDefined)
+      case totalPax if totalPax.exists(tp => tp.feedSource == AclFeedSource && tp.pax.isDefined) =>
+        excludeTransferPax(totalPax.find(tp => tp.feedSource == AclFeedSource))
       case totalPax =>
         totalPax.find(tp => tp.feedSource == UnknownFeedSource)
     }
-    matchedTotalPax.getOrElse(TotalPaxSource(0, UnknownFeedSource, None))
+    matchedTotalPax.getOrElse(TotalPaxSource(None, UnknownFeedSource))
+  }
+
+  def excludeTransferPax(totalPaxSource: Option[TotalPaxSource]) = {
```

Comment:
This isn't quite right. It's removing trans pax from each source and then setting it as the result as long as one of them is greater than zero, but it means if acl's result is greater than zero and live is negative we'll be using a negative number for the pax

Reply:
Would be good to write a test for this case